### ISSUE TITLE
Adding also sitelinks to Wikidata while reconciling

### ISSUE THREAD

**Issue by SannitaSSJ** [OpenRefine/OpenRefine#1788]
**Is your feature request related to a problem or area of OpenRefine? Please describe.**
It would be awesome to have a functionality that lets users add also sitelinks to Wiki* projects, through a Wikidata schema.

**Describe the solution you'd like**
Adding some space below the statements part in the Wikidata schema, that works kinda the label/description area, but that lets you add sitelinks to Wiki* projects.

**Describe alternatives you've considered**
As of now, there are no practical alternatives (except doing this _outside_ the OR framework.

**Comment by VojtechDostal** [comment]
Yes this would be lovely. I missed the sitelink functionality once or twice already

**Comment by wetneb** [comment]
As part of this it could be useful to update Wikidata-Toolkit's API to make it easier to add / delete sitelinks. This is currently only possible by sending the whole item JSON to the server, whereas we are currently using incremental changes with the `updateTermsStatements` method:
https://github.com/Wikidata/Wikidata-Toolkit/blob/master/wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WikibaseDataEditor.java

**Comment by VojtechDostal** [comment]
For anyone that is looking for a solution:
Wikibase-Cli started supporting this: https://github.com/maxlath/wikibase-cli/blob/main/docs/write_operations.md

**Comment by trnstlntk** [comment]
I have received a new request for this feature just today.

Status: unfortunately there are no plans or people/money to work on this yet. 

As a circumvention, I personally add sitelinks by working on an OpenRefine project, then export the schema to QuickStatements, and add the sitelinks by adding a column for them to the QuickStatements csv.
